File: Clase-laptop/Laptop.py
class Laptop:
    # Constructor
    def __init__(self, marca, modelo, ram, almacenamiento,precio):
        self.marca = marca
        self.modelo = modelo
        self.ram = ram
        self.almacenamiento = almacenamiento
        self.precio=precio

    def mostrar_informacion(self):
        print(f"Marca: {self.marca}")
        print(f"Modelo: {self.modelo}")
        print(f"Memoria RAM: {self.ram} GB")
        print(f"Almacenamiento: {self.almacenamiento} GB")
        print(f"Precio: $ {self.precio}")


# Crear un objeto de la clase Laptop
mi_laptop = Laptop("Lenovo", "ThinkPad", 16, 512,42000000)

mi_laptop.mostrar_informacion()

File: Clase-laptop/Laptop.java
public class Laptop {

    String marca;
    String modelo;
    int ram;
    int almacenamiento;
    double precio;

    public Laptop(String marca, String modelo, int ram, int almacenamiento,double precio) {
        this.marca = marca;
        this.modelo = modelo;
        this.ram = ram;
        this.almacenamiento = almacenamiento;
        this.precio=precio;
    }


    public void mostrarInformacion() {
        System.out.println("Marca: " + marca);
        System.out.println("Modelo: " + modelo);
        System.out.println("Memoria RAM: " + ram + " GB");
        System.out.println("Almacenamiento: " + almacenamiento + " GB");
        System.out.println("Precio: $" + precio);
    }

    public static void main(String[] args) {

        Laptop miLaptop = new Laptop("Lenovo", "ThinkPad", 16, 512,42000000);

        miLaptop.mostrarInformacion();
    }
}